lib/wsdl.js

```diff
@@ -1558,7 +1558,11 @@ WSDL.prototype.objectToXML = function(obj, name, namespace, xmlns, first, xmlnsA
       }
 
       if (!Array.isArray(child)) {
-        parts.push(['<', ns, name, attr, xmlnsAttrib, '>'].join(''));
+        var insNull = '';
+        if(typeof child === 'object' && child === null) {
```

Comment:
You're correct, I'm sure I added it for a reason, let me verify it in my tests! And if I can't figure out why then I'll get rid of it :) Thanks for your feedback, appreciate it!